src/testing/euler_naca0012_optimization.cpp

```diff
@@ -550,7 +550,7 @@ ::optimize (const unsigned int nx_ffd, const unsigned int poly_degree) const
     dealii::VectorTools::interpolate(dg->dof_handler, initial_conditions, dg->solution);
     // Create ODE solver and ramp up the solution from p0
     std::shared_ptr<ODE::ODESolverBase<dim, double>> ode_solver = ODE::ODESolverFactory<dim, double>::create_ODESolver(dg);
-    ode_solver->n_refine = 0;
+    //ode_solver->n_refine = 0;
```

Comment:
Instead of commenting it out. Just delete. Git saves this information.

Reply:
Deleted it.